docs/css-prop.md

```diff
@@ -6,8 +6,24 @@ The primary way to style elements with emotion is the `css` prop. It provides a
 
 ## Get Started
 
+There are 2 ways to get started with the `css` prop.
+
+#### Add `@emotion/babel-preset-css-prop` to babel configuration to enable for entire project
+
```

Comment:
Maybe this should say it doesn't work in CRA because some people might not know that CRA doesn't allow babel configuration and might not read the bit below

Reply:
Good call.